lib/Runtime/Library/JavascriptObject.cpp

```diff
@@ -2060,4 +2060,195 @@ namespace Js
         }
         return returnValue;
     }
+
+	/*static*/
+	void JavascriptObject::CopyDataPropertiesHelper(Var source, RecyclableObject* to, PropertyId* excluded, uint32 excludedLength, ScriptContext* scriptContext)
+	{
+		// 1. Assert Type(to) is Object.
+		// 2. Assert Type(excluded) is List.
+		// 3. If from is undefined or null, let keys be a new empty List
```

Comment:
Should this `from` actually be `source`?

Reply:
yes, updated

Reply:
I originally had different variable names and forgot to update the comment after changing...